### DISCUSSION TITLE
Import for custom fields at Components

### DISCUSSION THREAD

**Discussion by tim-hafner** [snipe/snipe-it#11797] [General]
Hi there,
is there any option to import components with custom fields into Snipe?
The import wanted for components:

Item Name | Model No. | TariffNumber | ID-Nr. | Quantity | Einzelpreis | Notes | Company | Order Number | Purchase Date | Supplier

Did I miss something or is this not possible with the actual Snipe version?
Kind regards
Tim

**Comment by corydlamb** [comment]
Does this model asset have this fieldset included in its model page? If so you should just be able to import it by referencing to it by name in the CSV like you've done here.
